examples/long_poll_example/README.md

```diff
@@ -0,0 +1,137 @@
+long polling example
+---------------------
+
+In this example, we show how you can perform long polling of resources using curl.
+
+> Note: etcd must be running (on localhost:4001 by default). You can change this in gohan.yaml.
+
+long polling a list
+--------------------
+
+First, perform normal GET request:
+
+```
+curl localhost:9091/v1.0/store/pets/ -i
+```
+
+Example response:
+
+```
+HTTP/1.1 200 OK
+Etag: "7b2f681a074dc89a5c918eb9ddbfec79"
+...
+{
+  "pets": []
+}
+```
+
+In the response headers, you will notice ``Etag``:
+
+```
+Etag: "7b2f681a074dc89a5c918eb9ddbfec79"
+```
+
+The string in quotes will be different depending on response content.
+Now, in a separate terminal, start a long polling GET request of list of resources, by providing a ``Long-Poll`` header with value of just received ``Etag`` header:
+
+```
+curl localhost:9091/v1.0/store/pets/ -H 'Long-Poll: "7b2f681a074dc89a5c918eb9ddbfec79"' -i
+```
+
+This command should block. In another terminal create a new resource:
+
+```
+curl http://localhost:9091/v1.0/store/pets/ -H "Content-Type: application/json" -X POST -d '{"name":"fluffy"}'  -i
+```
+
+Notice that the long polling request returned, and that it contains a new ``Etag``:
+
+```
+HTTP/1.1 200 OK
+Etag: "45169fff532c04c4ea721c04467771d6"
+...
+{
+  "pets": [
+    {
+      "description": null,
+      "id": "7d941380-7ca5-43bc-9246-7020ed7efcaf",
+      "name": "fluffy",
+      "status": null,
+      "tenant_id": "admin"
+    }
+  ]
+}
+```
+
+This new ``Etag`` value can be used to perform new long polling request.
+
+> Note: long polling requests will return upon any modifying operation on queried resource, which can be creation, deletion or modification.
+
+long polling single resource
+-----------------------------
+
+Notice, how long polling ``/v1.0/store/pets/`` path returns when a child object is created.
```

Comment:
I don't get meaning of this sentence.

Developers can watch a List API and a Show API. If a developer watch List API, Gohan server returns diff on list request result. If a developer watch Show API, Gohan server returns update on a single resource. 

Is this what you meant?